src/controllers/scripting/legacy/controllerscriptinterface.cpp

```diff
@@ -120,7 +120,7 @@ void ControllerScriptInterface::setValue(
 
     if (coScript != nullptr) {
         ControlObject* pControl = ControlObject::getControl(
-                coScript->getKey(), ControlFlag::NoAssertIfMissing);
+                coScript->getKey(), ControlFlag::AllowMissingOrInvalid);
```

Comment:
@Holzhaus please check that I merged the recent changes correctly.

Reply:
Looks okay, but you should apply the changes from #3056, too. Basically just replace the hardcoded flag with this function: https://github.com/mixxxdj/mixxx/pull/3056/files#diff-9289ba7fe799f555352e9653196f4b00R33-R39

Reply:
Reimplemented in 427ba5e